runtime-modules/project-token/src/events.rs

```diff
@@ -1,17 +1,30 @@
-use crate::types::{TokenDataOf, TransferPolicyOf, Transfers, Validated};
+use crate::types::{
+    JoyBalanceOf, RevenueSplitId, TokenIssuanceParametersOf, TokenSaleId, TransferPolicyOf,
+    Transfers, Validated,
+};
 use frame_support::decl_event;
 use sp_runtime::Perquintill;
 
 decl_event! {
     pub enum Event<T>
     where
         Balance = <T as crate::Trait>::Balance,
+        BlockNumber = <T as frame_system::Trait>::BlockNumber,
+        JoyBalance = JoyBalanceOf<T>,
         TokenId = <T as crate::Trait>::TokenId,
         AccountId = <T as frame_system::Trait>::AccountId,
         TransferPolicy = TransferPolicyOf<T>,
-        TokenData = TokenDataOf<T>,
+        TokenIssuanceParameters = TokenIssuanceParametersOf<T>,
         ValidatedTransfers = Transfers<Validated<<T as frame_system::Trait>::AccountId>, <T as crate::Trait>::Balance>,
     {
+
+        /// Token amount is unreserved
+        /// Params:
+        /// - token identifier
+        /// - account tokens are unreserved from
```

Comment:
> No other term is allowed: liquidity, reserved, reservability, free balance, usability... its all banished!

_Source: https://github.com/Joystream/joystream/issues/3633_

Based on this, please restrain from using terms like `reserved` / `unreserved`. I suggest using `unstaked` instead.